tests/io_utils.go

```diff
@@ -89,30 +89,27 @@ func CreateSCSIDisk(nodeName string, opts []string) (address string, device stri
 	ExpectWithOffset(1, err).NotTo(HaveOccurred(), "Failed to create faulty disk")
 
 	EventuallyWithOffset(1, func() error {
-		args = []string{UsrBinVirtChroot, Mount, Proc1NsMnt, "exec", "--", "/usr/bin/lsscsi"}
+		args = []string{UsrBinVirtChroot, Mount, Proc1NsMnt, "exec", "--", "/bin/sh", "-c", "/bin/grep -l scsi_debug /sys/bus/scsi/devices/*/model"}
```

Comment:
the `/bin/sh -c` is probably not needed.

Reply:
It's so the * is expanded by the shell

Reply:
Ah, makes sense.